addons/event/models/event.py

```diff
@@ -259,6 +265,18 @@ def mail_attendees(self, template_id, force_send=False, filter_func=lambda self:
         for attendee in self.registration_ids.filtered(filter_func):
             self.env['mail.template'].browse(template_id).send_mail(attendee.id, force_send=force_send)
 
+    @api.multi
+    def notify_schedule_updation_link(self):
+        return {
+            'type': 'ir.actions.act_window',
+            'name': 'Attendees',
+            'view_type': 'list',
```

Comment:
you added new view_type? we have tree value for list view in view_type for action ;) same for view_mode.